src/components/OutfitDetails/OutfitDetails.js

```diff
@@ -53,17 +53,17 @@ const OutfitDetails = ({ user, setAppError, appError, closeMenu}) => {
     setNewPieces(prev => [...prev, piece.id]);
   }
 
-  const deleteWarning = (e) => {
-    e.target.nextElementSibling.showModal()
+  const deleteWarning = () => {
+    document.querySelector('.delete-warning').showModal()
   }
 
-  const deleteOutfit = (e) => {
+  const deleteOutfit = () => {
     deleteData('outfits', user.userID, {id: outfitID});
     setDeleteSuccess(true);
     pieces.forEach(piece => {
       deleteData('outfit-to-pieces', user.userID, {outfitID, pieceID: piece.id})
     })
-    e.target.parentElement.parentElement.parentElement.close()
+    document.querySelector('.delete-warning').close()
```

Comment:
Wow Im sorry I literally forgot JavaScript for a second and forgot query selectors was a thing